content/docs/installation/operator-lifecycle-manager.md

```diff
@@ -7,7 +7,7 @@ description: 'cert-manager installation: Using OLM'
 
 ### Prerequisites
 
-- Install a [supported version of Kubernetes or OpenShift](./supported-releases.md).
+- Install a [supported version of Kubernetes or OpenShift](../releases/).
```

Comment:
suggestion: (from standup) I think I'd expect this to link to ../releases/README.md ?

Reply:
fixed in https://github.com/cert-manager/website/pull/1317/commits/8f82a55e231e27158c0abd8577ee72d0c51cf3eb